compiler-plugin/src/main/java/io/ballerina/lib/np/compilerplugin/NPFunctionValidator.java

```diff
@@ -18,17 +18,14 @@
 
```

Comment:
Did we consider the import prefix scenario

Reply:
Yes. Will update the logic to handle this as well.

Reply:
Fixed